ci/lint/check-bazel-buildifier.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# Checks bazel buildifier format
+
+set -euxo pipefail
+
+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE:-$0}")" || exit; pwd)/.."
```

Comment:
I am not sure if there are really people that will attempt to run these lint script locally.

Reply:
probably no-one